### ISSUE TITLE
some libs are not found (can render system unbootable after install)

### ISSUE THREAD

**Issue by ghost** [util-linux/util-linux#52]
I downloaded this ftp://ftp.kernel.org/pub/linux/utils/util-linux/v2.24/util-linux-2.24.1.tar.gz
unpacked it, did (EDIT: `./configure` was first, obviously) `make` then `sudo make install`  tested if `blkdiscard` works, seems to work, read it's `man` then later on rebooted but then `mount` wouldn't work... 

this is what `umount` says ('cause I already fixed `mount` by then)

```
umount: /lib/x86_64-linux-gnu/libmount.so.1: version `MOUNT_2.25' not found (required by umount)
umount: /lib/x86_64-linux-gnu/libmount.so.1: version `MOUNT_2.23' not found (required by umount)
umount: /lib/x86_64-linux-gnu/libmount.so.1: version `MOUNT_2.22' not found (required by umount)
umount: /lib/x86_64-linux-gnu/libmount.so.1: version `MOUNT_2.21' not found (required by umount)
umount: /lib/x86_64-linux-gnu/libmount.so.1: version `MOUNT_2.24' not found (required by umount)
```

and lsblk:

```
lsblk: /lib/x86_64-linux-gnu/libmount.so.1: version `MOUNT_2.22' not found (required by lsblk)
lsblk: /lib/x86_64-linux-gnu/libmount.so.1: version `MOUNT_2.24' not found (required by lsblk)
```

this is under linux mint 15 Olivia 64bit

took a while to fix to be able to get back in 'cause of other issues that added to this; 
I actually didn't realize it(the `sudo make install`) replaced the existing `mount` and a lot of others, I'm not really sure how to restore the originals, that is, I'd have to find out in which package are each of the replaced commands that fail to run and then from X run synaptic and set them for Reinstall (seems to have worked for the `mount` package so far)

Basically it's my fault here, I should've been more careful and not just `sudo make install` so eagerly. ;)

Peaces

**Comment by StefanFabian** [comment]
First, I have to say follow @falconindy's advice and don't do that.
However, if you (like me) already did this, here's how I fixed it (on Ubuntu 16.04):
If you didn't reboot yet, skip to step 4:

1. Boot into a live cd (if you can't boot your system anymore)
2. Mount your stuff (this highly depends on your system but in default Ubuntu installs that's the following)
2.1 Mount your root partition to /mnt
2.2 Possibly mount your boot partition to /boot or your efi partition to /boot/efi
2.3 Rebind the interfaces:
     `for dir in /dev /dev/pts /proc /sys /run; do sudo mount --bind $dir /mnt/$dir; done `
3. Chroot to /mnt
`sudo chroot /mnt`


4. Now, go to the source folder of util-linux and uninstall it again using make uninstall
5. Reinstall it with the configuration from here (Simply follow the steps listed there) http://www.linuxfromscratch.org/lfs/view/development/chapter06/util-linux.html
6. Finally, to fix the double libraries cd to /lib/x86_64-linux-gnu/ and rename the existing libraries of util-linux there using i.e.
`for lib in libblkid.so* libfdisk.so* libmount.so* libsmartcols.so* and libuuid.so*; do mv $lib $lib.bak; done`

It might or better it's very likely that this left some traces and maybe broke some other stuff but in my case it boots again and seems to work like before again.
